WebView/app/src/main/assets/style.css

```diff
@@ -91,4 +85,20 @@ img {
   padding-top: 20%;
   padding-left: 5%;
   padding-right: 5%;
+}
+@media (prefers-color-scheme: dark) {
+  body {
+    background: #383838;
+  }
+  h1 {
+    color: white;
+  }
+  h2 {
```

Comment:
nit: maybe reformat this as

```css
h1, h2 {
  // ...
}
```

for stylistic consistency with the light theme CSS